praw/models/reddit/subreddit.py

```diff
@@ -882,6 +882,12 @@ def edited(self, only=None, **generator_kwargs):
         Additional keyword arguments are passed in the initialization of
         :class:`.ListingGenerator`.
 
+        To print all items in the edited queue try:
+        .. code:: python
+
+          for item in reddit.subreddit('mod').mod.edited(limit=None):
+            print(item)
```

Comment:
For the PRAW documentation can you please insure that new scopes are indented with 4 spaces relative to its parent scope?